### PR TITLE
Add sonata media bundle and configure

### PR DESCRIPTION
**Pull request by Etheriq**

### CODE REVIEW

File: app/config/sonata_projects_configuration.yml [hunk 1]
```diff
@@ -0,0 +1,96 @@
+sonata_admin:
+    title:      Cherkassy Dream Admin Panel !
+    templates:
+        ## default global templates
+        layout:  SonataAdminBundle::standard_layout.html.twig
+        ajax:    SonataAdminBundle::ajax_layout.html.twig
+
+        ## default actions templates, should extend a global templates
```
Comment (spolischook) [inline app/config/sonata_projects_configuration.yml]: please don't recobfigure default values

File: app/config/sonata_projects_configuration.yml [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+    entity_manager: ~
+
+    templates:
+        form:
+            - SonataDoctrineORMAdminBundle:Form:form_admin_fields.html.twig
+        filter:
+            - SonataDoctrineORMAdminBundle:Form:filter_admin_fields.html.twig
+        types:
+            list:
+                array:      SonataAdminBundle:CRUD:list_array.html.twig
```
Comment (spolischook) [inline app/config/sonata_projects_configuration.yml]: please don't recobfigure default values

File: src/Application/Sonata/MediaBundle/ApplicationSonataMediaBundle.php [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+class ApplicationSonataMediaBundle extends Bundle
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function getParent()
+    {
+        return 'SonataMediaBundle';
+    }
+}
```
Comment (spolischook) [inline src/Application/Sonata/MediaBundle/ApplicationSonataMediaBundle.php:32]: Don't forget for empty line

File: web/upload/upl.txt [hunk 4]
```diff
@@ -0,0 +1 @@
+.
```
Comment (spolischook) [inline web/upload/upl.txt]: I think upload directory  must be in .gitignore

File: src/Application/Sonata/MediaBundle/ApplicationSonataMediaBundle.php (hunk 3 above)
Comment (spolischook) [inline src/Application/Sonata/MediaBundle/ApplicationSonataMediaBundle.php:32]: If it generator do this, leave as is.

Comment (Etheriq) [inline src/Application/Sonata/MediaBundle/ApplicationSonataMediaBundle.php:32]: yes, it did generator

**Pull request merged**